components/mpas-ocean/cime_config/buildnml

```diff
@@ -207,6 +207,17 @@ def buildnml(case, caseroot, compname):
             logger.warning("WARNING: The specified compset is requesting ocean ICs spunup from a G-case")
             logger.warning("         But no file available for this grid.")
 
+    elif ocn_grid == 'ARRM10to60E2r1':
+        decomp_date = '220730'
+        decomp_prefix = 'mpas-o.graph.info.'
+        restoring_file = 'sss.PHC3.0_monthlyClimatology.ARRM10to60E2r1.220802.nc'
+        analysis_mask_file = 'ARRM10to60E2r1_mocBasinsAndTransects220730.nc'
```

Comment:
I don't think this file should have been given the date it was created.  The date stamp on the `mocBasinsAndTransects` files is a date stamp for the regions used in the masks, rather than the masks themselves.  This won't matter in E3SM but it will be a problem if the file to be added to`diagnostics` was modified in this way.  In general, I think it would be better to leave the file with the date stamp created in `compass`:
```
ARRM10to60E2r1_mocBasinsAndTransects20210623.nc
```

Reply:
For a little more background, this date is controlled and updated in `geometric_features`:
https://github.com/MPAS-Dev/geometric_features/blob/master/geometric_features/aggregation/__init__.py#L56-L58

Reply:
Thanks for pointing this out. I just diffed the file with the `220730` datestamp and the `20210623` one and they are the same, so I must have `mv`ed this to match the other datestamps? I shouldn't have done that - I'll fix it.

Reply:
we did go through a couple of mesh iterations between June and July though..

Reply:
> we did go through a couple of mesh iterations between June and July though..
@milenaveneziani, that date stamp is from June 2021, not 2022.  It is the date that we added the separate Atlantic and Atlantic+Mediterranean MOC basins to geometric features and has nothing to do with when the file or the mesh itself was created.